peer/network.go

```diff
@@ -135,16 +138,16 @@ func (n *network) Request(nodeID ids.NodeID, request []byte, responseHandler mes
 	n.lock.Lock()
 	defer n.lock.Unlock()
 
-	return n.request(nodeID, request, responseHandler)
+	return n.sendRequest(nodeID, request, responseHandler)
 }
 
-// request sends request message bytes to specified nodeID and adds [responseHandler] to [outstandingRequestHandlers]
+// sendRequest sends request message bytes to specified nodeID and adds [responseHandler] to [outstandingRequestHandlers]
 // so that it can be invoked when the network receives either a response or failure message.
 // Assumes [nodeID] is never [self] since we guarantee [self] will not be added to the [peers] map.
 // Releases active requests semaphore if there was an error in sending the request
 // Returns an error if [appSender] is unable to make the request.
 // Assumes write lock is held
-func (n *network) request(nodeID ids.NodeID, request []byte, responseHandler message.ResponseHandler) error {
+func (n *network) sendRequest(nodeID ids.NodeID, request []byte, responseHandler message.ResponseHandler) error {
```

Comment:
again can we rename this to sendAppRequest

Reply:
yup